bin/node/runtime/Cargo.toml

```diff
@@ -176,6 +176,7 @@ runtime-benchmarks = [
 	"pallet-treasury/runtime-benchmarks",
 	"pallet-utility/runtime-benchmarks",
 	"pallet-vesting/runtime-benchmarks",
+	"pallet-chainbridge/runtime-benchmarks",
```

Comment:
Why do we need this line here? Revert plz

Reply:
Without this build will success but test will fail. @evgeny-s